broker-daemon/proto/broker.proto

```diff
@@ -925,6 +925,11 @@ message ReleaseChannelsRequest {
   bool force = 2;
 }
 
+message ReleaseChannelsResponse {
+  // Array of errors that may have occurred when releasing a specific market
+  repeated string errors = 1;
```

Comment:
it seems like we ought to be returning more than just an array of strings. Perhaps a list of messages related to each engine? or to each channel? with success or failure.